velox/dwio/catalog/fbhive/test/FileUtilsTests.cpp

```diff
@@ -68,6 +71,6 @@ TEST(FileUtilsTests, ExtractPartitionName) {
   for (auto testCase : testCases) {
     EXPECT_EQ(
         testCase.partitionName,
-        FileUtils::extractPartitionName(testCase.filePath));
+        FileUtils::extractPartitionName(testCase.filePath, true));
```

Comment:
I wonder if we need this new parameter here. I assume not, but if we do need it, then we should add tests for both true and false values.